ui/css/treeherder.css

```diff
@@ -1221,7 +1221,14 @@ ul.failure-summary-list li .btn-xs {
     border-color: rgba(70, 70, 70, 0.25);
 }
 
-.result-status-shading-coalesced {background-color: white;}
+.result-status-shading-unknown {
+    background-color: rgb(252, 248, 227);
+    border-color: rgb(251, 216, 144);
```

Comment:
I used the rgb equivalent of our hex colors for the new yellow, to match the rgba style of all the other `result-status-shading-*` entries. I am tempted to either: switch this one to hex, or perhaps provide the hex equivalent for context as an inline comment.